javascript/javascript1/week1/houseprice.js

```diff
@@ -0,0 +1,31 @@
+var housePrice;
+var volumeInMeters;
+var gardenSizeInM2;
```

Comment:
- There are many old things in JavaScript, everyone should know because you will find many old codes at the internet and at the job later. But you should try not to use them nowadays, one of them is `var`. Always use `const` and if you want to re-assign a new value later use `let`
- Try to declare your variables when you assign a value. What I mean do this
```js
const gardenSizeInM2 = 100;
```
not this
```js
var gardenSizeInM2;
gardenSizeInM2 = 100;
```

Reply:
Because the variable  'gardenSizeInM2' is used again for julias  house price , havent made it 'const'. I am not sure if I can do it.